### ISSUE TITLE
where "chr1_before_find_islands.csv" file comes from?

### ISSUE THREAD

**Issue by zhenyisong** [biocore-ntnu/epic#70]
I installed the local version epic through bioconda. However, when task was competed, there was a file named "chr1_before_find_islands.csv" which was created along with expected cvs files. Here is my shell snippet:
```
    macs2 callpeak --treatment ${treat} --control ${control} \
          --format BAM --gsize mm --name ${base} --bdg --qvalue 0.01
```

here is the excerpt of this file ("chr1_before_find_islands.csv")

```
 Chromosome Bin ChIP Input Score
28 chr1 3007400 6 5.0 2.43113976041
134 chr1 3035800 6 4.0 2.43113976041
174 chr1 3046800 6 1.0 2.43113976041
177 chr1 3047400 6 3.0 2.43113976041
```
I am confused about this unwanted file. It seems that the run_epic.py will output this file. (at line 60)

```
dfs[0].to_csv("chr1_before_find_islands.csv", sep=" ")
```

Will you please clarify this chr1 output? Or it is just the temporary file and can be neglected whatever. Or the error message we should take care and tune the parameter in the command line?

**Comment by endrebak** [comment]
Thanks for reporting. I really appreciate it. The file was used for
debugging and should not have been included in a release. Will fix tomorrow!

Endre

**Comment by endrebak** [comment]
This is fixed in 2.9 which is out on pypi but not bioconda yet. Will add it now :)
